src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorSpec.scala

```diff
@@ -45,7 +45,29 @@ class SubmissionMonitorSpec(_system: ActorSystem) extends TestKit(_system) with
     }
   }
 
-  ignore should "transition to running then completed then terminate" in withDefaultTestDatabase { dataSource =>
+  it should "transition nicely without actor shenanigans" in withDefaultTestDatabase { dataSource =>
```

Comment:
This test was supposed to be equivalent to the actor-ified version below, but it missed an entity save that happens in the middle :(